lib/utils/getServerlessConfigFile.js

```diff
@@ -10,18 +10,24 @@ const getServerlessConfigFile = _.memoize((servicePath) => {
   const jsonPath = path.join(servicePath, 'serverless.json');
   const ymlPath = path.join(servicePath, 'serverless.yml');
   const yamlPath = path.join(servicePath, 'serverless.yaml');
+  const jsPath = path.join(servicePath, 'serverless.js');
 
   return BbPromise.props({
     json: fileExists(jsonPath),
     yml: fileExists(ymlPath),
     yaml: fileExists(yamlPath),
+    js: fileExists(jsPath),
   }).then((exists) => {
     if (exists.json) {
       return readFile(jsonPath);
     } else if (exists.yml) {
       return readFile(ymlPath);
     } else if (exists.yaml) {
       return readFile(yamlPath);
+    } else if (exists.js) {
+      // use require to load serverless.js
+      // eslint-disable-next-line global-require
+      return require(jsPath);
```

Comment:
`require` can throw synchronously. You're in a promise chain here, so you should catch any error and return a promise rejection in case it fails and you must return a resolved promise if it suceeds.